src/accessibility-statement.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Accessibility statement
+description: accessibility in the GOV.UK Design System
+layout: layout-single-page-prose.njk
+---
+
+# Accessibility statement
+
+The GOV.UK Design System team (the team) is committed to making its website at [design-system.service.gov.uk](https://design-system.service.gov.uk/) and the GOV.UK Frontend documentation website at [frontend.design-system.service.gov.uk](https://frontend.design-system.service.gov.uk/) accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+
+This accessibility statement applies to both websites and the components and patterns from the GOV.UK Frontend codebase which appear in the examples throughout the Design System.
+
+The team wants to make sure as many people as possible can use their websites. For example, making sure the website text is as clear as possible to understand and making sure you can:
+
+-   change colours, contrast levels and fonts
+-   zoom in up to 400% without the text spilling off the screen
+-   listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA, TalkBack and VoiceOver)
+-   navigate most of the website using just a keyboard
+-   navigate most of the website using just a mouse and on-screen keyboard
+-   navigate most of the website using speech recognition software
+
+AbilityNet has [advice on making your device easier to use](https://mcmw.abilitynet.org.uk/), if you have a disability.
+
+## Compliance status
+
+The Design System website at [design-system.service.gov.uk](https://design-system.service.gov.uk/) is fully compliant with the Web Content Accessibility Guidelines (WCAG) version 2.1 AA standard.
+
+The GOV.UK Frontend documentation website at [frontend.design-system.service.gov.uk](https://frontend.design-system.service.gov.uk/) is fully compliant with the Web Content Accessibility Guidelines (WCAG) version 2.1 AA standard.
+
+The GOV.UK Frontend codebase at [github.com/alphagov/govuk-frontend](https://github.com/alphagov/govuk-frontend) is fully compliant with the Web Content Accessibility Guidelines (WCAG) versions 2.1 AA and 2.2 AA standards.
```

Comment:
Hi @dav-idc, @stevenjmesser wanted to see if we can just say "2.2 AA" to keep this sentence simple, as it's inclusive of 2.1 AA?

Reply:
Hey @calvin-lau-sig7 ! Yep, while 2.2 isn't entirely inclusive of 2.1 (they deprecated success criterion 4.1.1 Parsing), for our purposes and from a practical perspective 2.2 is backwards compatible and replaces 2.1. 👍

Reply:
Thanks, appreciated, nearly there!

```suggestion
The GOV.UK Frontend codebase at [github.com/alphagov/govuk-frontend](https://github.com/alphagov/govuk-frontend) is fully compliant with the Web Content Accessibility Guidelines (WCAG) version 2.2 AA standard.
```